src/OSPSuite.Infrastructure/Services/OSPSuiteLogger.cs

```diff
@@ -3,19 +3,22 @@
 
 namespace OSPSuite.Infrastructure.Services
 {
-   public class OSPLogger : IOSPSuiteLogger
+   public class OSPSuiteLogger : IOSPSuiteLogger
    {
+      public string DefaultCategoryName { get; set; }
+
       protected const string DEFAULT_LOGGER_CATEGORY = "OSPSuite";
       private readonly ILoggerCreator _loggerCreator;
 
-      public OSPLogger(ILoggerCreator loggerCreator)
+      public OSPSuiteLogger(ILoggerCreator loggerCreator, string defaultCategoryName = DEFAULT_LOGGER_CATEGORY)
```

Comment:
Since we are instantiating this guy with container, you will never be able to pass a default category name. So I'd remove the constructor unless you need it for testing?